src/unitxt/error_utils.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from .logging_utils import get_logger
+
+logger = get_logger()
+
+DOCUMENTATION_URL = "https://www.unitxt.ai/en/latest/"
+DOCUMENTATION_HUGGINGFACE_METRICS = "docs/adding_metric.html#adding-a-hugginface-metric"
+DOCUMENTATION_ADDING_TASK = "docs/adding_task.html"
```

Comment:
```suggestion
class Documentation:
    base = "https://www.unitxt.ai/en/latest/"
    HUGGINGFACE_METRICS = "docs/adding_metric.html#adding-a-hugginface-metric"
    ADDING_TASK = "docs/adding_task.html"
```
Maybe more neat and useful in @benjaminsznajder constants class style?

Then you have auto complete and only one import per file

Reply:
Good idea. Done.